jekyll/_cci2/orb-concepts.md

```diff
@@ -247,7 +247,12 @@ _[See: Writing Inline Orbs]({{site.baseurl}}/2.0/reusing-config/#writing-inline-
 ## Private orbs vs. public orbs
 {: #private-orbs-vs-public-orbs }
 
-There are two ways to publish an orb: publin or private. If you prefer to publish your orb internally, and not to the [CircleCI Orb Registry](https://circleci.com/developer/orbs), you will want to use a private orb. However, if you want to publish your orb to the [CircleCI Orb Registry](https://circleci.com/developer/orbs), use a public orb. Each type of orb is described in the sections below.
+There are two ways to publish an orb: public or private: 
```

Comment:
This is just a personal preference, but I think something like "There are two ways to publish an orb, publicly or privately." sounds better. I think something about the double colon wigs me out 🙃

Reply:
Yeah it looks bad, I don't know how that happened tbh!